include/aikido/rviz/TSRMarker.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef AIKIDO_RVIZ_TSRMARKER_H_
+#define AIKIDO_RVIZ_TSRMARKER_H_
+
+#include <set>
+#include "FrameMarker.hpp"
+#include "SmartPointers.hpp"
+
+namespace aikido {
+namespace rviz {
+
+class TSRMarker {
+public:
+  TSRMarker();
+  ~TSRMarker(){};
+
+  void addFrameMarker(FrameMarkerPtr const &marker);
+
+private:
+  std::set<FrameMarkerPtr> mFrameMarkers;
```

Comment:
Nit: Why does this need to be an `std::set`? I'd prefer an `std::vector` here, since linear memory is a lot simpler than a red-black tree, if we don't require uniqueness.